docs/workflows/workflow-retry-configuration.md

```diff
@@ -0,0 +1,48 @@
+### Retry Configuration
+In the RetryPassWorkflow, the HelloWorld step has an attribute labeled `Retry`. Valid __json__ configuration of this attribute for workflows is explained in the AWS [documentation](https://docs.aws.amazon.com/step-functions/latest/dg/amazon-states-language-errors.html#amazon-states-language-retrying-after-error). Cumulus deployment (kes) requires configuration to be written in (or translated to) `.yml` format, but required attributes and their functions are the same.
```

Comment:
I would say something like 'Configuring retries on a task allows for the task to be retried to handle transient issues such as those arising from resource allocation or throttling instead of failing the entire workflow. Cumulus supports the AWS retry configuration outlined here.'

Reply:
that is really well put. I'm probably going to steal 98% of it